AUTHORS

```diff
@@ -36,7 +36,9 @@ Mindy Preston <mindy.preston@docker.com>
 Natanael Copa <natanael.copa@docker.com>
 Nathan Dautenhahn <ndd@seas.upenn.edu>
 Nathan LeClaire <nathan.leclaire@docker.com>
+Nico Di Rocco <dirocco.nico@gmail.com>
 Phil Estes <estesp@linux.vnet.ibm.com>
+radu-matei <matei.radu94@gmail.com>
```

Comment:
Oh, that needed an entry in `.mailmap` so that it automatically maps the e-mail to the preferred name. Let me do a follow-up